doc/classes/AABB.xml

```diff
@@ -126,7 +126,8 @@
 			<return type="Vector3" />
 			<param index="0" name="dir" type="Vector3" />
 			<description>
-				Returns the support point in a given direction. This is useful for collision detection algorithms.
+				Returns the vertex of the AABB that's the farthest in a given direction.
+				This point is commonly known as the support point in collision detection algorithms.
```

Comment:
```suggestion
				Returns the vertex of the AABB that's the farthest in a given direction. This point is commonly known as the support point in collision detection algorithms.
```
I would go even further and make the entire thing one line.

In the XML docs, every line break is a new paragraph.

Reply:
Ah my apologies, I didn't realize line breaks cause a new paragraph